packages/mobile/src/exchange/ExchangeTradeScreen.test.tsx

```diff
@@ -1,6 +1,8 @@
+// @ts-ignore
+import { toBeDisabled } from '@testing-library/jest-native'
```

Comment:
Why is `@ts-ignore` needed here?

Reply:
If I remove the // @ts-ignore I receive the warning the following warning in the IDE; however the tests run without issue.
`Module '"@testing-library/jest-native"' has no exported member 'toBeDisabled'.ts(2305)`